src/screens/login/index.tsx

```diff
@@ -1,25 +1,54 @@
-import React from 'react';
+import React, { FC, useState } from 'react';
+
+import { login, registration } from '../../api/userAPI';
 
 import Background from '../../components/background/Background';
 import Title from '../../components/title/Title';
 
 import { Icons } from '../../assets/Icons';
 import { Colors } from '../../constants/colors';
 import { TemplateType } from '../../constants/templateType';
+import { UserType } from '../../types/types';
 
 import { ButtonText, LoginFrame, LoginInput, SignInButton, SignUpButton } from './styles';
 
-export const Login = () => {
+export const Login: FC = () => {
+  const [email, setEmail] = useState();
+  const [password, setPassword] = useState();
+  const [user, setUser] = useState<UserType>();
```

Comment:
somewhere you specify the type of the state and somewhere you dont. Either specify it everywhere or don't specify it. Also it's not UserType, it's UserType | undefined.